generators/client/templates/angular/src/main/webapp/app/layouts/error/error.route.ts.ejs

```diff
@@ -24,25 +24,17 @@ export const errorRoute: Routes = [
   {
     path: 'error',
     component: ErrorComponent,
-    data: {
-      pageTitle: 'error.title',
-    },
+    title: 'error.title',
```

Comment:
What about being more explicit "title.key"

Reply:
I don't understand :/

Reply:
Ah, sorry
```
{
    path: 'error',
    component: ErrorComponent,
    title: { key:...,
}}
```

Reply:
Would be interesting, but I don't think it's possible.
[https://angular.io/api/router/TitleStrategy `buildTitle](https://angular.io/api/router/TitleStrategy#buildtitle) returns a string.